### ISSUE TITLE
Hard to reproduce result

### ISSUE THREAD

**Issue by wenhuchen** [kimiyoung/review_net#1]
Hello,

I ran your baseline system with weight sharing and without discriminative loss, I only got BLEU-4 around 25.6 instead of 28.2 reported in the paper. May I ask you how many epochs did you train your model ? I trained it for 3-4 epochs and then it converges somehow.

**Comment by kimiyoung** [comment]
It seems that you didn't do beam search? What's your command to run the code?

I tried it myself and it can reproduce the exact numbers.

**Comment by wenhuchen** [comment]
Oh, I see ! I saw this score from printout information during training on the validation set, I didn't use reason_att_eval.lua to evaluate, I will try it out now.
